tools/vagrant/Vagrantfile

```diff
@@ -92,57 +93,68 @@ Vagrant.configure('2') do |config|
         fi
 
         cd /home/vagrant/openwhisk/ansible
-        su vagrant -c 'ansible-playbook -i environments/local setup.yml'
-        su vagrant -c 'ansible-playbook -i environments/local prereq.yml'
+        su vagrant -c 'ansible-playbook -i environments/vagrant setup.yml'
+        su vagrant -c 'ansible-playbook -i environments/vagrant prereq.yml'
```

Comment:
currently, the `-i environments/local` is used for vagrant installations.  will that continue to work with this new vagrant installation/environment?